### PR TITLE
Ie/translate file trip element

### PR DESCRIPTION
**Pull request by UnLowkee**

### CODE REVIEW

**Review by ThibautHERR — APPROVED** [review]

File: src/app/modules/features/points/components/points-list/points-list.component.html [hunk 1]
```diff
@@ -43,9 +43,8 @@ <h4 class="text-align-center m-0">{{'Points' | translate}}</h4>
         >
           <spt-point-preview [point]="point" [isHighlighted]="(highlightedPointId$ | async) === point.id">
             <div class="d-flex">
-              <!-- todo: Translate nbTooltip -->
               <button nbButton ghost (click)="editPoint(point)"
-                nbTooltip="Modifier le point d'intérêt et ses documents."
+                nbTooltip='UpdateInterestPointAndDocuments' | translate
```
Comment (Augustin68) [inline src/app/modules/features/points/components/points-list/points-list.component.html:47]:
```suggestion
                [nbTooltip]="('UpdateInterestPointAndDocuments' | translate)"
```

File: src/app/modules/features/step/components/steps-list/steps-list.component.html [hunk 2]
```diff
@@ -33,9 +33,8 @@ <h4>{{'Steps' | translate}}</h4>
             [isHighlighted]="(highlithedStepId$ | async) === step.stepInstance.id"
           >
             <div>
-              <!-- todo: TRANSLATE nbTooltip -->
               <button nbButton ghost (click)="editStep(step.stepInstance)"
-                nbTooltip="Modifier l'étape et ses documents"
+                nbTooltip='UpdateInterestPointAndDocuments' | translate
```
Comment (Augustin68) [inline src/app/modules/features/step/components/steps-list/steps-list.component.html:37]:
```suggestion
                [nbTooltip]="('UpdateInterestPointAndDocuments' | translate)"
```

File: src/app/modules/shared/components/trip-element-details/trip-element-details.component.html [hunk 3]
```diff
@@ -3,10 +3,10 @@
   <h2>{{pageTitle}}</h2>
 
   <nb-tabset class="d-flex flex-column justify-content-center">
-    <nb-tab tabTitle="Modifier" tabIcon="edit-outline" >
+    <nb-tab tabTitle="('Modify' | translate)" tabIcon="edit-outline" >
```
Comment (Augustin68) [inline src/app/modules/shared/components/trip-element-details/trip-element-details.component.html:6]:
```suggestion
    <nb-tab [tabTitle]="('Modify' | translate)" tabIcon="edit-outline" >
```

File: src/app/modules/shared/components/trip-element-details/trip-element-details.component.html [hunk 4]
```diff
@@ -3,10 +3,10 @@
...
       <ng-content select="[edit]"></ng-content>
     </nb-tab>
-    <nb-tab tabTitle="Documents" tabIcon="file-text-outline">
+    <nb-tab tabTitle="('TitleDocuments' | translate)" tabIcon="file-text-outline">
```
Comment (Augustin68) [inline src/app/modules/shared/components/trip-element-details/trip-element-details.component.html:9]:
```suggestion
    <nb-tab [tabTitle]="('TitleDocuments' | translate)" tabIcon="file-text-outline">
```

**Review by Augustin68 — CHANGES_REQUESTED** [review]
You need to put brackets around element directives passed as variables

**Pull request merged**